Providers/Profile.swift

```diff
@@ -210,6 +210,23 @@ open class BrowserProfile: Profile {
 
     weak fileprivate var app: UIApplication?
 
+    let db: BrowserDB
+    let loginsDB: BrowserDB
+
+    private static var loginsKey: String? {
+        let key = "sqlcipher.key.logins.db"
+        let keychain = KeychainWrapper.sharedAppContainerKeychain
+        if keychain.hasValue(forKey: key) {
+            let value = keychain.string(forKey: key)
+            return value
+        } else {
```

Comment:
No need for `else` after return. No need for `let` then immediate return, either.